### PR TITLE
Move chapter owners to page maintainers

### PR DESCRIPTION
**Pull request by bouweandela**
- [x] I followed the [CONTRIBUTING guidelines](../blob/main/CONTRIBUTING.md).

Add a maintainer to every page that we plan to keep (see https://github.com/NLeSC/guide/issues/286 for a list).

### CODE REVIEW

File: best_practices/language_guides/opencl_cuda.md [hunk 1]
```diff
@@ -65,3 +65,6 @@
  * Generic Auto Tuners:
      * [Kernel Tuner](https://github.com/benvanwerkhoven/kernel_tuner) (Python)
      * [CLTune](https://github.com/CNugteren/CLTune) (C++)
+
+
+*Page maintainer: unmaintained*
```
Comment (bouweandela) [inline best_practices/language_guides/opencl_cuda.md:70]: @benvanwerkhoven Would you still be interested in maintaining the OpenCL/CUDA page now that you have a new place to work?

Comment (benvanwerkhoven) [reply best_practices/language_guides/opencl_cuda.md:70]: Hi Bouwe! Indeed, I think it might be better if someone else, for example @isazi, takes over.

Comment (isazi) [reply best_practices/language_guides/opencl_cuda.md:70]: Hi @bouweandela I will take over that from Ben. Let me know what else you need from me.

**Comment by egpbos** [comment]
What do you think about putting the maintainers on top of the page instead to make them a bit more visible? We're going to be asking them periodically for some work, so I feel it's fair to give them some clear proper credit.

File: best_practices/language_guides/opencl_cuda.md (hunk 1 above)
Comment (bouweandela) [reply best_practices/language_guides/opencl_cuda.md]: Thanks @isazi! Being a page maintainer involves checking once a year or so if the content is still up to date (and updating it if it turns out to be outdated) and if there is a pull request from someone else you will be asked to review it.

**Comment by bouweandela** [comment]
> What do you think about putting the maintainers on top of the page instead to make them a bit more visible?

Done in [ca79bf1](https://github.com/NLeSC/guide/pull/297/commits/ca79bf1f9d0d90110058a349a66d5916bf72b7b3)

**Review by egpbos — APPROVED** [review]
LGTM! We should probably add maintainers to almost all pages (except the front page maybe), but that can be a separate PR.

**Comment by bouweandela** [comment]
Agreed. I added them to all pages that we plan to keep, though I may have missed some.

**Pull request merged**